test/cases/expanded/zero argument macro.c

```diff
@@ -0,0 +1,10 @@
+
+
+
+
+
+
+
+
+NO_ARGUMENTS(1)
```

Comment:
Both GCC and clang remove the argument list, implementing that behavior should be as easy as copying the logic from the `error.Unterminated` prong from the switch above.